ee/app/livechat-enterprise/client/route.js

```diff
@@ -0,0 +1,58 @@
+import { AccountBox } from '../../../../app/ui-utils';
+import { livechatManagerRoutes } from '../../../../app/livechat/client/route';
+
+AccountBox.addRoute({
```

Comment:
The Livechat Enterprise routes are being added without license verification, unlike other modules, like this:
https://github.com/RocketChat/Rocket.Chat/pull/16944/files#diff-81b0ad498000ae05840d29ce6caf4fddR6

The License verification is only inside the Templates, due to that we can face something like this:

![Screen Shot 2020-03-20 at 12 17 41](https://user-images.githubusercontent.com/59577424/77178102-64b5b700-6aa5-11ea-9369-f881afa4c227.png)

I don't think it's the expected behaviour, am I right?
Why do we have different approaches among the Enterprise modules?

Reply:
we can think later how the best approach for this.. we could even add the menu options but show a message about that feature being enterprise only, for example.. having decided the best approach, we can implement everywhere 😉